__tests__/app.test.js

```diff
@@ -66,4 +67,45 @@ describe('2. GET /api/topics', ()=>{
             expect(msg).toBe('Not Found');
         })
     });
+});
+
+describe('3. GET /api/articles', ()=>{
+    test('responds with an array of article objects', ()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body: {articles}})=>{
+            expect(articles).toBeInstanceOf(Array);
+            expect(articles).toHaveLength(12);
+            articles.forEach((article)=>{
+                expect(article).toEqual(
+                expect.objectContaining({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(String)
+                })
+                );
+            });
+        });
```

Comment:
Would love an assertion to make sure 'body' is not present in the returned list of articles. In the ticket we don't explicitly say "don't have it" but it is implied, we don't want a big bulky chunk of body for each item coming back.